File: app/src/main/java/hr/amaurov/niamu/orm_presentation/orm/greenDao/models/Country.java
package hr.amaurov.niamu.orm_presentation.orm.greenDao.models;

import org.greenrobot.greendao.annotation.Entity;
import org.greenrobot.greendao.annotation.Id;

@Entity(
    // Flag if the DAO should create the database table (default is true).
    // Set this to false, if you have multiple entities mapping to one table,
    // or the table creation is done outside of greenDAO.
    createInDb = false,

    // Whether an all properties constructor should be generated.
    // A no-args constructor is always required.
    generateConstructors = true,

    // Whether getters and setters for properties should be generated if missing.
    generateGettersSetters = true
)
public class Country {
    @Id(autoincrement = true)
    private Long id;

    private String name;
}

File: app/src/main/java/hr/amaurov/niamu/orm_presentation/orm/room/entities/Country.kt
package hr.amaurov.niamu.orm_presentation.orm.room.entities

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName="countries")
data class CountryRoom (
        @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name="countryID")
    var countryID: Long? = null,
        @ColumnInfo(name="name")
    var name: String? = null
)